File: algo-practice/mixFruit-may-16-17.py
def mix_fruit(arr):
    '''Returns the price of a cup of juice as the mean of price
     of chosen fruits. The output price should be an integer.'''
    regular = ['banana', 'orange', 'apple', 'lemon', 'grapes']
    special = ['avocado', 'strawberry', 'mango']
    total = 0
    for fruit in arr:
        if fruit.lower() in regular:
            total += 5
        elif fruit.lower() in special:
            total += 7
        else:
            total += 9
    return round(total/len(arr))

File: algo-practice/mix_fruit-may16-17.js
function mix_fruit(arr) {
	let regular = ['banana', 'orange', 'apple', 'lemon', 'grapes'];
    let special = ['avocado', 'strawberry', 'mango'];
    let total = 0;
    for (let i = 0; i < arr.length; i++) {
    	if (regular.indexOf(arr[i].toLowerCase()) >= 0) {
    		total += 5;
    	}
    	else if (special.indexOf(arr[i].toLowerCase()) >= 0) {
    		total += 7;
    	}
    	else {
    		total += 9;
    	}
    }
    return Math.round(total/arr.length);
}